Snakefile

```diff
@@ -131,7 +131,7 @@ rule run_tbprofiler:
         eval "$(conda shell.bash hook)"
         conda activate tb-profiler
         scripts/run_tbprofiler.sh {wildcards.sample} \
-        {params.s3_bucket} \
+        "{params.s3_bucket}" \
```

Comment:
Looks like the quotes were added to make sure the positional arguments still match even if the `s3_bucket` is an empty string. I'd be worried about someone tripping over this later... 

I'd update the script to move the `s3_bucket` to be a last optional argument:

```bash
sample="$1"
tb_output_path="$2"
fastq_outdir="$3"
tb_outdir="$4"
threads="$5"
s3_bucket="${6:-}"
```
Then you wouldn't need quotes here

```
scripts/run_tbprofiler.sh {wildcards.sample} \
{params.tb_output_path} \
{params.fastq_outdir} \
{params.tb_outdir} \
{threads} \
{params.s3_bucket} \
```